experimental/dds/tree2/src/test/feature-libraries/sequence-field/sequenceChangeRebaser.spec.ts

```diff
@@ -162,22 +188,17 @@ describe("SequenceField - Rebaser Axioms", () => {
 								);
 								const r1 = rebaseTagged(change1, change2);
 								tracker.apply(change2);
-								normalizeMoveIds(r1.change);
 								const r2 = rebaseTagged(r1, inverse2);
 								tracker.apply(inverse2);
 								// We need to update change2 to ensure it refers to detached nodes by the detach
 								// that last affected them.
-								const change2Updated = tracker.update(
-									change2,
-									continuingAllocator([change2]),
-								);
+								// TODO: This should not be necessary, as in a real sandwich rebase, this step would not happen.
```

Comment:
Can you expand on what currently makes it necessary?

Reply:
I need to look into this more, but it's quite possible this will be fixed by adding lineage to inverses.